Resources/Prototypes/species.yml

```diff
@@ -33,7 +33,7 @@
 - type: species
   id: Skeleton
   name: Skeleton
-  roundStart: true #touch this an feel my wrath
+  roundStart: true #touch this and feel my wrath
```

Comment:
```suggestion
  roundStart: false #touch this and feel my wrath | bet --moony
```

Reply:
Moony wasnt maintainer conduct talked about today during the meeting?
(Ps i am in your walls)